modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,144 @@
+====
+
+.Procedure
+
+. Using the Administrator perspective on the web console, update any Operator Lifecycle Manager (OLM) Operators to the versions that are compatible with the version you are updating to. You can find more information on how to perform this action on the link:https://docs.openshift.com/container-platform/4.12/operators/admin/olm-upgrading-operators.adoc[Updating installed Operators] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
+To view the status of all machine config pools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. Change to the `eus-4.12` channel by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.12
+----
++
+[NOTE]
+====
+
+If you receive an error message indicating that `eus-4.12` is not one of the
+available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS updates.
+This rollout process generally takes 45-90 days starting at the GA date.
+====
++
+
+. Pause all worker machine pools except for the master pool by running the following command:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+----
+
+. Update to the latest version by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.11.18
+----
+
+. Review the cluster version to ensure that the updates are complete by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.11.18   True       False         6m29s   Cluster version is 4.11.18
+----
+
+. Update to version 4.12 by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. Retrieve the cluster version to ensure that the 4.12 updates are complete by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.12.1   True       False         6m29s   Cluster version is 4.12.1
+----
+
+. Update your worker nodes to 4.12 by unpausing all previously paused machine config pools. Once your worker nodes have been updated, run the following command:
```

Comment:
the flowing sentence is incorrect:
`Once your worker nodes have been updated, run the following command` 
since at this point the cluster is updated, but the workers are not even started

Reply:
changed to: "To update your worker nodes to 4.12, unpause all previously paused machine config pools by running the following command:"